dist/stat_data.py

```diff
@@ -282,8 +282,13 @@ def __init__(self, name, desc, flags=''):
     CacheStat('cache_lookaside_insert', 'cache overflow table insert calls'),
     CacheStat('cache_lookaside_ondisk', 'cache overflow table on-disk size', 'no_clear,no_scale,size'),
     CacheStat('cache_lookaside_ondisk_max', 'cache overflow table max on-disk size', 'no_clear,no_scale,size'),
+    CacheStat('cache_lookaside_read', 'cache overflow table update reads'),
+    CacheStat('cache_lookaside_read_miss', 'cache overflow table update reads missed'),
+    CacheStat('cache_lookaside_read_squash', 'cache overflow table update reads requiring modify squashing'),
```

Comment:
This is tracking how often we have to read multiple entries from lookaside and squash them into a value. This happens to any modify operation that gets lookaside evicted so this shouldn't be that uncommon.